analysis/export/CSVExporter/src/test/kotlin/de/maibornwolff/codecharta/exporter/csvexporter/ParserDialogTest.kt

```diff
@@ -0,0 +1,45 @@
+package de.maibornwolff.codecharta.exporter.csvexporter
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptInput
+import com.github.kinquirer.components.promptInputNumber
+import de.maibornwolff.codecharta.exporter.csv.CSVExporter
+import de.maibornwolff.codecharta.exporter.csv.ParserDialog
+import io.mockk.every
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import picocli.CommandLine
+import java.io.File
+import java.math.BigDecimal
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ParserDialogTest {
+
+    @AfterAll
+    fun afterTest() {
+        unmockkAll()
+    }
+
+    @Test
+    fun `should output arguments that are parsed correctly`() {
```

Comment:
This is a test for the happy path. Do we have another one for the "negative" path. For example if the user specifies a string instead of a number for the third (compressing) question.

Reply:
It is not possible to specify a string when prompted for a number. That is handled from the Kotlin Inquirer library when using `promtNumberInput(...)` instead of `promtInput(...)`

Reply:
What is happening, if you specify a wrong number (e.g. a string)? I would ensure that the parser does not crash in this case and for example that the parsers asks again for the correct number. It this is already the case, then that's fine for me.